chapter_02_repository.asciidoc

```diff
@@ -489,7 +489,7 @@ trying to ask: "what do we get for this?  And what does it cost us?".
 
 Usually at the very least we'll be introducing an extra layer of abstraction,
 and although we may hope it will be reducing complexity overall, it does add
-complexity locally, and it has a cost in terms raw numbers of moving parts and
+complexity locally, and it has a cost in terms of moving parts and
```

Comment:
nice

Reply:
or maybe just lose the word "raw"

Reply:
Keeping the numbers would look wierd, I think the edit as is, is good